sympy/core/add.py

```diff
@@ -978,11 +985,12 @@ def as_real_imag(self, deep=True, **hints):
         return (self.func(*re_part), self.func(*im_part))
 
     def _eval_as_leading_term(self, x, cdir=0):
-        from sympy import expand_mul, Order
+        from sympy import Order
+        from .function import _mexpand
 
         old = self
 
-        expr = expand_mul(self)
+        expr = _mexpand(self)
```

Comment:
When does `expand_mul` not suffice?

Reply:
I had interpreted a result wrong, `expand_mul` does suffice

Reply:
Right... it is for this case I had mentioned on gitter - 
```py
In [22]: -3*x + (x + S(3)/2 - sqrt(3)*I/2)**2 - S(3)/2 + 3*sqrt(3)*I/2
Out[22]: 
                     2             
       ⎛    3   √3⋅ⅈ⎞    3   3⋅√3⋅ⅈ
-3⋅x + ⎜x + ─ - ────⎟  - ─ + ──────
       ⎝    2    2  ⎠    2     2   

In [23]: _.nseries(x)
Out[23]: 
                                  2              2         
  6⋅x      6⋅√3⋅ⅈ⋅x            6⋅x       6⋅√3⋅ⅈ⋅x      ⎛ 6⎞
──────── - ──────── - 3⋅x + ────────── - ────────── + O⎝x ⎠
3 - √3⋅ⅈ   3 - √3⋅ⅈ         6 - 6⋅√3⋅ⅈ   6 - 6⋅√3⋅ⅈ        

In [24]: _.as_leading_term(x)
Out[24]: 
-3⋅x⋅(1 + √3⋅ⅈ) 
────────────────
    3 - √3⋅ⅈ
```
So should `_mepxand` be kept? The series module tests all pass with `expand_mul` also.